packages/osd-ui-framework/Gruntfile.js

```diff
@@ -91,7 +92,7 @@ module.exports = function (grunt) {
           }
 
           postcss([postcssConfig])
-            .process(result.css, { from: src, to: dest })
+            .process(strip(result.css.toString('utf8')), { from: src, to: dest })
```

Comment:
I considered making this type of change in https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4644 - nice to see it here, although mostly I'm excited to be able to drop this whole package soon.